src/VirtualClient/VirtualClient.Main/profiles/DEPENDENCY-AMD-GPU-DRIVER.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Description": "AMD GPU Driver Installation Dependency",
+    "Metadata": {
+        "RecommendedMinimumExecutionTime": "00:10:00",
+        "SupportedPlatforms": "win-x64",
+        "SupportedOperatingSystems": "Windows",
+        "SupportedVmSeries":  "nvv4,v620",
+        "SpecialRequirements": "This is AMD GPU Driver dependency. It can only be installed on the system having an AMD GPU card/chip."
+    },
+    "Parameters": {        
+        "VmSeries": "nvv4"
```

Comment:
Let's not put anything Azure related. We should call this mi25, and this should work for all mi25 machine, host or vm